integrations/tensorflow/e2e/slim_vision_models/iree_slim_vision_test_suite.bzl

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Macro for building e2e keras vision model tests."""
+
+load("//bindings/python:build_defs.oss.bzl", "iree_py_test")
+load("@bazel_skylib//lib:new_sets.bzl", "sets")
+
+def iree_slim_vision_test_suite(
+        name,
+        model_tags,
+        backends,
+        reference_backend,
+        failing_configurations = None,
+        tags = None,
+        deps = None,
+        size = "large",
+        python_version = "PY3",
+        **kwargs):
+    """Creates a test for each configuration and bundles a succeeding and failing test suite.
+
+    Creates one test per model_tag and backend. Tests indicated in
+    `failing_configurations` are bundled into a suite suffixed with "_failing"
+    tagged to be excluded from CI and wildcard builds. All other tests are
+    bundled into a suite with the same name as the macro.
+
+    Args:
+      name:
+        name of the generated passing test suite. If failing_configurations is
+        not `None` then a test suite named name_failing will also be generated.
+      model_tags:
```

Comment:
Is "tags" the right word here. It's used to mean something different in Bazel, so a bit confusing

Reply:
That's how Oleg wrote it, so I assumed it was a Slim thing, but I can't find much info. Let's go with `models` to match the other vision macro?

Reply:
Done.